static/css/restyle.less

```diff
@@ -711,24 +717,24 @@ button.good {
     background: 0;
 
     .hovercard.addon {
-         &:hover,
-         &:hover:before,
-         &:hover .icon,
-         &:hover .summary,
-         &:hover .more {
-            background: none;
-            border-color: transparent;
-            box-shadow: none;
-         }
-
-         .more {
-             display: block !important;
-             .addon-summary,
-             .byline,
-             .vitals {
-                display: none;
-             }
+      &:hover,
+      &:hover:before,
+      &:hover .icon,
+      &:hover .summary,
+      &:hover .more {
+        background: none;
+        border-color: transparent;
+        box-shadow: none;
+      }
+
+      .more {
+         display: block !important;
```

Comment:
That's the historical indent size - but I'd be cool with change everything to 2 spaces. 4 looks huge to me but then I'm looking at JS most of the time which has 2 so it's not that surprising.